profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyBusinessDisplay.html

```diff
@@ -25,7 +25,7 @@
 <body>
 <wicket:panel>
 
-	<div class="mainSection editable">
+	<div class="mainSection editable" tabindex=0>
```

Comment:
so this needs to be selected in order to edit?

Reply:
I'm not sure how this was interned to work. With a mouse you have to hover over the div for the Edit button to appear and there way no way to make the button appear with the keyboard. I'm not sure what the editable was here.

I feel like the better design wouldn't be to have the hidden/popup Edit boxes and just have them be always in the UI. If we did that on this Profile page it would make it easier for a11y but I don't know where this original design came from.

Reply:
The design came from a desire for coolness. It was cool at the time when @steveswinsburg  did it. That's my defence of Swinny, my then understudy :)

We are both to blame!